packages/components/src/mobile/bottom-sheet/stepper-cell/stepper.android.js

```diff
@@ -47,8 +47,8 @@ function Stepper( {
 					isMinValue ? { opacity: 0.4 } : null,
 				] }
 			>
-				<Dashicon
-					icon="arrow-down-alt2"
+				<Icon
+					icon={ chevronDown }
```

Comment:
I was wondering if this change is correct. cc @Tug

Reply:
Just did a quick test, they do look a bit smaller in size:
![image](https://user-images.githubusercontent.com/230230/74024368-ffe54800-49a1-11ea-92a2-2cd612f7cd5f.png)

The svg path is different so I guess that's expected. I don't think the difference makes a big deal imo.

Reply:
This is likely because the footprint of the icons have increased from 20x20 to 24x24.

This is noteworthy as all icons in the package, going forward, will have a 24x24 footprint.